otwcode/otwarchive
# AO3-5554 Fix ruby version syntax inside .ruby-version

# Pull Request Checklist

* [ ] Have you added tests for any changed functionality? **N/A**

## Issue

https://otwarchive.atlassian.net/browse/AO3-XXXX (please fill in issue number and remove this comment)

## Purpose

This PR fixes an issue in the `.ruby-version`. That file previously specified the version as `ruby 2.3.4`, but `rbenv` (and perhaps others) expect `ruby-2.3.4`. See the ticket.

## Testing Instructions

This can be tested by `cd /vagrant/otwarchive` and `gem install bundler`. It should work, rather than complaining `rbenv: version `ruby' is not installed`. I am using the vagrant environment with `rvm`, so I have not tested this myself. Given that it is a pretty small change, and seems to be [exactly the change called for in the issue](https://otwarchive.atlassian.net/browse/AO3-5554), I'm hopeful it will work as intended
